drivers/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ add_subdirectory_ifdef(CONFIG_PM_CPU_OPS pm_cpu_ops)
 add_subdirectory_ifdef(CONFIG_FLASH_HAS_DRIVER_ENABLED flash)
 add_subdirectory_ifdef(CONFIG_SERIAL_HAS_DRIVER serial)
 add_subdirectory_ifdef(CONFIG_BT_DRIVERS       bluetooth)
-add_subdirectory_ifdef(CONFIG_NETWORKING       net)
+add_subdirectory_ifdef(CONFIG_NET_DRIVERS      net)
```

Comment:
Should you select `NET_DRIVERS` inside `subsys/net/Kconfig`

Reply:
Where exactly would you do that ?

If you take a sample like `bluetooth/ipsp` it will enable networking in its `prj.conf`:
https://github.com/zephyrproject-rtos/zephyr/blob/c8c3d32a0f126e449924fc3cbe0ed5d7d3cbadfa/samples/bluetooth/ipsp/prj.conf#L8

but it will not select any net drivers (loopback/slip/ppp) to enter `zephyr_library()` created here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/net/CMakeLists.txt

Resulting in:
```
  No SOURCES given to Zephyr library: drivers__net

  Excluding target from build.
```

That's why `CONFIG_NET_DRIVERS=y` must explicitly be selected when needed.